src/components/Selfie/style.css

```diff
@@ -0,0 +1,79 @@
+@import (less) "../Theme/constants.css";
+
+.thinWrapper {
+  margin: 0 39*@unit;
+}
+
+.introIcon {
+  display: block;
+  position: absolute;
+  left: 0;
+  top: 0;
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 48*@unit;
+  height: 48*@unit;
+}
+
+.introBullet {
+  padding-left: 48*@unit;
+  margin-bottom: 48*@unit;
+  /* list-style: none; */
+  position: relative;
+  text-align: left;
+
+  /* blue line connecting all three icons */
+  &::before {
+    content: '';
+    display: block;
+    background-color: @color-text-dark;
+    width: 2*@unit;
+    height: 150%;
+    position: absolute;
+    top: 24*@unit;
+    left: 23*@unit;
+  }
+
+  &:last-child::before {
+    display: none;
+  }
+}
+
+.introBullets {
+  padding-left: 0;
+  margin-top: 41*@unit;
+  display: table;
+  list-style: none;
+}
+
+.introText {
+  padding-left: 17*@unit;
+  -webkit-box-decoration-break: clone;
+  box-decoration-break: clone;
+  height: 48*@unit;
+  display: table-cell;
+  vertical-align: middle;
+}
+
+.selfieIcon {
+  background-image: url('assets/selfie-picto.svg');
+  background-size: contain;
+}
+
+.glassesIcon {
+  background-image: url('assets/glasses-picto.svg');
+  background-size: contain;
+}
+
+.introCopy {
+  margin-bottom: auto;
+}
+
+.lineSeparator {
+  border-left: 6px solid green;
+  height: 500px;
+  position: absolute;
+  left: 50%;
+  margin-left: -3px;
+  top: 0;
+}
```

Comment:
I don't think this class is in use as I don't see any green lines/borders on the Intro screen

Reply:
deleted